packages/datadog-plugin-kafkajs/src/consumer.js

```diff
@@ -23,10 +31,46 @@ class KafkajsConsumerPlugin extends ConsumerPlugin {
       metrics: {
         'kafka.partition': partition
       }
-    })
+    }
+
+    if (this.config.dsmEnabled) {
```

Comment:
One thing you can do to decrease the amount of indentation is to use an early return. Totally optional and up to you.

```js
    if (!this.config.dsmEnabled) {
      return
    }
    // remaining code is 1 level shallower 
```

Reply:
nvm, this is wrong in this case